spec/index.bs

```diff
@@ -7305,49 +7293,49 @@ All packed texture formats support {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsa
             <th>Extension
     </thead>
     <tr>
-        <td data-componenttype=float>{{GPUTextureFormat/rgb9e5ufloat}}
+        <td>{{GPUTextureFormat/rgb9e5ufloat}}
```

Comment:
I assume we should add a component type column to this table?

Reply:
There is already one as the third column.